financial-services/1.8/modules/ROOT/pages/sfdc-fsc-setup-guide.adoc

```diff
@@ -609,8 +589,7 @@ Follow the below steps to create the Apex Triggers
 === Additional Notes
 
 * All the Apex Triggers created above filters records based on `LastModifiedBy <> {Service account ID}`. This is to filter out updates done by the Salesforce Customers and Salesforce Financial System APIs to avoid cyclical updates. This can be obtained by running the query `SELECT Id FROM User WHERE Username='<service-account-username>'` in the Developer Console.
-* The Apex Triggers AccountEventsPub, and AccountAddressEventsPub also filter records based on record type ID. The record type ID can be obtained by running the query `SELECT Id,Name FROM RecordType where Name='Household' and sObjectType='Account'` in the Developer Console.
-* The Apex Trigger HouseholdEventsPub also filter records based on record type ID. The record type ID can be obtained by running the query `SELECT Id,Name FROM RecordType where Name='Account' and sObjectType='Account'` in the Developer Console.
+* The Apex Triggers AccountEventsPub, AccountAddressEventsPub, and HouseholdEventsPub filter records based on record type ID. A query is povided in the respective Apex trigger as a comment and it can be used to obtain the record type ID. 
```

Comment:
```suggestion
* The Apex Triggers AccountEventsPub, AccountAddressEventsPub, and HouseholdEventsPub filter records based on record type ID. A query is provided in the respective Apex trigger as a comment and it can be used to obtain the record type ID. 
```